app/assets/javascripts/creator.js

```diff
@@ -1,5 +1,3 @@
 Blacklight.onLoad( function() {
-	var authoritySelect = require('authority_select');
-	var as = new authoritySelect.AuthoritySelect({ selectBox : "select#work_creator", inputField : "input.work_creator" });
-	as.initialize();
+	Hyrax.authoritySelect({ selectBox : "select#work_creator", inputField : "input.work_creator" });
```

Comment:
It's not the duplicaion that's the problem, it's that you haven't specified an id parameter to one of them, so both use the default.